src/Surfnet/StepupMiddleware/ApiBundle/Configuration/Service/AllowedSecondFactorListService.php

```diff
@@ -60,4 +61,16 @@ public function getAllowedSecondFactorMap()
     {
         return AllowedSecondFactorMap::from($this->allowedSecondFactorRepository->findAll());
     }
+
+    /**
+     * @param SecondFactorType $secondFactor
+     * @param Institution $institution
+     * @return bool
+     */
+    public function isSecondFactorAllowedFor(SecondFactorType $secondFactor, Institution $institution)
```

Comment:
what is the purpose of this method? Why is a decision, that is deliberately exposed in a VO, hidden in a service? I would have expected the `AllowedSecondFactorList` to be retrieved with the "getter" and then used for the decision. This seems to hide the complexity for no apparent gain? Thoughts?